Pyroscope/Pyroscope/Pyroscope.csproj

```diff
@@ -2,13 +2,14 @@
 
     <PropertyGroup>
```

Comment:
I am not super familiar with `csproj`, but as our profiler is only working on `linux/amd64`, is there a way to limit this very explicitly?

Reply:
I would not recommend doing that as it would break compilation for consumers on other operating systems/architectures.

I use this package in an application that is _deployed_ as an Ubuntu container, but it supports being developed on Linux, macOS and Windows. The build is configured to only install the profiler and setup Pyroscope in the appropriate environment.